client/src/App.tsx

```diff
@@ -1,8 +1,17 @@
-import React from "react";
 import logo from "./logo.svg";
 import "./App.css";
+import { useQuery } from "urql";
+import { GetHistogramsDocument, GetHistogramsQuery } from "./generated";
+
+const App = () => {
+  // Test query to make sure connection with graphql endpoint works
+  // TODO: Remove this block once we have a working integration test
+  const [result] = useQuery<GetHistogramsQuery>({
+    query: GetHistogramsDocument,
```

Comment:
So the `GetHistogramsQuery` works and you see the listHistograms give you a list. However it seems on the GetHistogramsDocument query I get

```
result 
    Object
        data: undefined
        error: undefined
        extensions: undefined
        fetching: true
        operation: undefined
        stale: false
[[Prototype]]: Object
```
Not sure if it's a syntax issue on the query (like not asking for fields back/inputting arguments) or if I need to make a change to my schema to support something you expected to see

Reply:
Resolved: Doug just got confused with re-renders because he is new to React